<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Server</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="ipText">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portText">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>123</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>开启</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="showText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
